docs/book/src/user/quick-start.md

```diff
@@ -7,10 +7,51 @@ In this tutorial we'll cover the basics of how to use Cluster API to create one
 - Install and setup [kubectl] in your local environment.
 - A Kubernetes cluster to be used as [management cluster].
   For the purpose of this quick start we suggest you to use [kind] as a _non production-ready_ solution.
+
+{{#tabs name:"kind-cluster" tabs:"AWS,Docker,vSphere"}}
+{{#tab AWS}}
+
+  ```bash
+  kind create cluster --name=clusterapi
+  export KUBECONFIG="$(kind get kubeconfig-path --name="clusterapi")"
+  ```
+{{#/tab }}
+{{#tab Docker}}
+
+<aside class="note warning">
+
+<h1>Not Intended For Production</h1>
+
+Because it requires no external infrastructure, the Docker provider allows you to quickly try Cluster API. It is also useful for developing Cluster API, and tooling that uses it. **Running a production cluster using the Docker provider is not supported**.
+
+</aside>
+
+  Because the Docker provider needs to access Docker on the host, a custom kind cluster configuration is required:
+
+  ```bash
+  cat > kind-cluster-with-extramounts.yaml <<EOF
+kind: Cluster
+apiVersion: kind.sigs.k8s.io/v1alpha3
+nodes:
+  - role: control-plane
+    extraMounts:
+      - hostPath: /var/lib/docker
+        containerPath: /var/lib/docker
```

Comment:
Is `/var/lib/docker` actually needed here? My testing to date has just mounted `/var/run/docker.sock` only.

Reply:
Thanks. Removing.